Makefile

```diff
@@ -206,6 +207,18 @@ include graphics.mk
 chao_garden/mb_chao_garden.gba.lz: chao_garden/mb_chao_garden.gba 
 	$(GFX) $< $@ -search 1
 
+%/interactables.bin: %/interactables.csv
+	$(EPOS) $< $@ -entities INTERACTABLES -header "./include/constants/interactables.h"
```

Comment:
Oh I didn't realise you can give it the constants as a header, ok that's cool! :)